core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -104,6 +104,11 @@ class ConsumerBounceTest extends IntegrationTestHarness with Logging {
     consumer.subscribe(tp)
     consumer.seek(tp, 0)
 
+    // wait until all the followers have synced the last HW with leader
```

Comment:
@guozhangwang That makes sense, thanks for the explanation. But is it not also possible that the writes could get lost in a scenario like that of KAFKA-1211 where the log gets truncated to the HW after a second leader failure? That would also reveal itself in the first seekToEnd returning an earlier than expected offset.